SPECS/ca-certificates-prebuilt/ca-certificates-prebuilt.spec

```diff
@@ -0,0 +1,42 @@
+Summary:        Prebuilt version of ca-certificates package
+Name:           ca-certificates-prebuilt
+
+Version:        20200720
```

Comment:
Maybe also add comment saying, that this should always be kept aligned with the version of `ca-certificates`.

Reply:
Changed my mind, new comment in a new file.

---
In reply to: [538808680](https://github.com/microsoft/CBL-Mariner/pull/403#discussion_r538808680) [](ancestors = 538808680)